tooling/FST-1030-FSharp.Core nuget package netstandard 2.0 only.md

```diff
@@ -0,0 +1,59 @@
+
+Subsequently, FSharp.Core.dll has been recompiled for "trimmed down" platforms: portable profiles, Xamarin profiles, .NET Core, .NET Standard and .NET Framework. There is even a version of FSharp.Core for the "Xamarin iOS TV" profile.... When provided by Microsoft, these were signed, strong-named and installed under "Reference Assemblies" on Windows. For other F# compilation environments like Xamarin, Mono, Cloud Sharper, FSharp.Formatting, Azure Notebooks, Azure Functions there was a bit of a mess (made considerably worse by the separation of sigdata/optdata files, now addressed)
+
+The F# Core Engineering Group created a NuGet package called FSharp.Core, partly to avoid a proliferation of "homebrew" packages appearing at that time.  For the last 3 years management and distribution of this nuget package has occured at the fsharp OSS site: https://github.com/dotnet/fsharp
+
+Today
+For the last 3 years the package has been released as a signed nuget package supporting both net45+ and netstandard2.0 TFMs.
+
+## implementation
+The implementation is easy.  Today at the fsharp oss project site: https://github.com/dotnet/fsharp, the fsharp.core nuget package and signed dll's are built.  We will remove the build for the desktop clr and all multi-fsharp.core testing support.
+
+This will work well, because the Windows desktop and netstandard FSharp.Core dll's have identical public surfaces.  And net472 automagically modiifies references to netstandard to load the specific net4X version of the api.
+
+## Assumptions
+For the purposes of this RFC we will assume that:
+
+# Scenarios
+Please contribute scenarios to the list below.
+
+1. General purpose library developer targetting usage on both NetCoreApp *, and net472+ frameworks uses F# to build an library.
+When the FSharp.Core targets netstandard 2.0  only then a nuget package with a single netstandard2.0 will work well, the developer will be able to build libraries that target all coreclr frameworks from netcoreapp2.0 up, and all desktop frameworks from net472 to net48.
+
+2. Application developer targetting windows development build a command line application  uses F# to build an application.
+The developer can target Windows using net472 and net48, or the coreclr with all netcoreapps from 2.0 up
+
+3. Application developer targetting linux / mac development build a command line application  uses F# to build an application.
+The developer can target Windows linux / mac or windows using the coreclr with all netcoreapps from 2.0 up
+
+4. Application developer targetting windows / linux / mac using net5 uses F# to build an app.
+The developer can target Windows linux / mac or windows using the coreclr with all net5 up
+
+5. General purpose library developer targetting usage on net45+ frameworks uses F# to build an library.
```

Comment:
We should probably mention that net461 works in most cases with NetStandard 2.0 libs, even though it's not a recommended scenario. See https://docs.microsoft.com/en-us/dotnet/standard/net-standard.

I'm not sure, and perhaps we should test it, but considering the limited amount of surface area that FSharp.Core touches, it's very well possible that it's compatible with net461 (and maybe even 4.5? as I believe there's currently a job that successfully runs in that version). 

I'm not saying we should support that in an official fashion, but I think we should mention it as a possibly working scenario for companies that cannot upgrade their .Net version yet.

Reply:
@abelbraaksma, the guidance from the dotnet team is net472 supports netstandard2.0, the others may work it is at the developers risk.  We are choosing to say pick an earlier FSharp.Core nuget package for supported downlevel frameworks.

Reply:
Fair enough. I just thought we should mention it, as the ms docs suggest it too.

Reply:
@abelbraaksma , I think the goal here is to be crisp about what is supported.

Reply:
Gotcha!